script/.build-deb/rules

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/make -f
+
+VERSION = $(shell cat VERSION)
+
+override_dh_gencontrol:
+	# if we're on Ubuntu, we need to Recommends: apparmor
+	echo 'apparmor:Recommends=$(shell dpkg-vendor --is Ubuntu && echo apparmor)' >> debian/runc.substvars
+	dh_gencontrol
+
+override_dh_auto_build:
+	make all
+	# ./man/md2man-all.sh runs outside the build container (if at all), since we don't have go-md2man here
+
+override_dh_auto_test:
+	./runc -v
+
+override_dh_strip:
+	# the SHA1 of dockerinit is important: don't strip it
+	# also, Go has lots of problems with stripping, so just don't
+
+override_dh_auto_install:
+	mkdir -p debian/runc/usr/bin
+	cp -aT "$$(readlink -f runc)" debian/runc/usr/bin/runc
+
+override_dh_installinit:
+	dh_installinit --name=runc
+
+override_dh_installudev:
+	# match our existing priority
+	dh_installudev --priority=z80
```

Comment:
I think we can get rid of both `override_dh_installudev` and `override_dh_installinit` since we're not shipping any udev files or init scripts here yet, right?